modules/end-user-guide/pages/basic-actions-you-can-perform-on-a-workspace.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: REFERENCE
+:description: Basic actions you can perform on a workspace
+:keywords: running-workspace, stopped-workspace, open-workspace, open-a-workspace, open-a-running-workspace, opening-workspace, opening-a-workspace, opening-a-running-workspace, how-to-open-workspace, how-to-open-a-workspace, how-to-open-a-running-workspace, reopen-workspace, reopen-a-workspace, reopen-a-running-workspace, reopening-workspace, reopening-a-workspace, reopening-a-running-workspace, how-to-reopen-workspace, how-to-reopen-a-workspace, how-to-reopen-a-running-workspace, restart-workspace, restart-a-workspace, restart-a-running-workspace, restarting-workspace, restarting-a-workspace, restarting-a-running-workspace, how-to-restart-workspace, how-to-restart-a-workspace, how-to-restart-a-running-workspace, stop-workspace, stop-a-workspace, stop-a-running-workspace,  stopping-workspace, stopping-a-workspace, stopping-a-running-workspace, how-to-stop-workspace, how-to-stop-a-workspace, how-to-stop-a-running-workspace, start-workspace, start-a-workspace, start-stopped-workspace, start-a-stopped-workspace, starting-workspace, starting-a-workspace, starting-stopped-workspace, starting-a-stopped-workspace, how-to-start-workspace, how-to-start-a-workspace, how-to-start-stopped-workspace, how-to-start-a-stopped-workspace, delete-workspace, delete-a-workspace, deleting-workspace, deleting-a-workspace, how-to-delete-workspace, how-to-delete-a-workspace
```

Comment:
"Keyword stuffing" de-ranks your content: https://developers.google.com/search/docs/advanced/guidelines/irrelevant-keywords

Reply:
Thanks for the heads up. In this case, all the keywords are descriptive of the actual content of the page. I ended up with such a long list because I previously had the subsections here as separate pages, where each page had its own list. I ended up "merging" the several pages into one page, and I just added up the keyboards.

The quantity of keyboards aside, my main motivator for adding keywords is making sure that the combinations that readers may type in the Search field in our docs pages pick up the relevant page.

Reply:
Fixed. Thank you both.